### PR TITLE
Adjust CODEOWNERS file to new reality

### PR DESCRIPTION
**Pull request by hurryabit**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

---
This change is [Reviewable](https://reviewable.io/reviews/digital-asset/daml/4293)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
```
Comment (cocreature) [inline CODEOWNERS]: Do I really have to be a codeowner? I very much like to be able to differentiate between people requesting my review explicitly which I get as email notifications and just codeowner notifications which I mostly get simply by looking at all PRs.

File: CODEOWNERS [hunk 2]
```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
...
+/libs-haskell/      @hurryabit @cocreature
+/daml-assistant/    @associahedron @hurryabit
```
Comment (cocreature) [inline CODEOWNERS]: If I do have to be a codeowner I probably should be a codeowner here as well given that I’ve written significant parts of `daml-helper` and the integration tests.

File: CODEOWNERS [hunk 3]
```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
...
+/daml-script/       @cocreature @hurryabit
```
Comment (cocreature) [inline CODEOWNERS]: @aherrmann-da should probably also be a code owner on this if we want to be accurate.

File: CODEOWNERS [hunk 4]
```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
...
+/triggers/          @cocreature @hurryabit
```
Comment (cocreature) [inline CODEOWNERS]: Same here.

**Comment by hurryabit** [comment]
@cocreature I added all your suggestions.

File: CODEOWNERS [hunk 5]
```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
+/libs-haskell/      @hurryabit @cocreature
+/daml-assistant/    @associahedron @hurryabit @associahedron
```
Comment (cocreature) [inline CODEOWNERS]:
```suggestion
/daml-assistant/    @associahedron @hurryabit @cocreature 
```

While @associahedron is certainly the ultimate head of DAML assistant, you probably wanted to add me.

Comment (cocreature) [reply CODEOWNERS]: Moved to https://github.com/digital-asset/daml/pull/4306

**Pull request merged**